Kickstarter-iOS/Views/Cells/DiscoveryPostcardCell.swift

```diff
@@ -257,21 +257,15 @@ internal final class DiscoveryPostcardCell: UITableViewCell, ValueCell {
 
     self.watchProjectViewModel.outputs.generateImpactFeedback
       .observeForUI()
-      .observeValues { [weak self] in
```

Comment:
We now no longer need `self` reference...in the future this could also be refactored into a side effect so there would not be a need for `generateImpactFeedback` signal from the VM. I identified that this would make the changes much bigger and decided against doing that.